src/sympc/tensor/additive_shared.py

```diff
@@ -1,172 +0,0 @@
-import torch
```

Comment:
The AddtiveSharingTensor becomes the ShareTensorCC(ShareTensorControlCenter -- it has a field with pointers to shares)